docs/src/operate/installation/init_systems.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Init Systems
+tags: [Installation, Unix, Init, Services]
+sidebar_position: 1
+---
+
+# Init systems
+
+thin-edge.io supports Systemd out of the box, however not all Linux distributions use Systemd. To use thin-edge.io on a Linux distribution without Systemd requires a few extra steps.
+
+Support for different init systems (service managers) is provided by a community repository, [tedge-services](https://github.com/thin-edge/tedge-services). The following service definitions are currently supported (though check the community repository if you don't see your preferred init system in the list).
+
+* OpenRC
+* runit
+* s6-overlay
+* SysVinit
+* supervisord
+
+If you want to understand how the thin-edge.io abstracts the init system, then check out the [init system reference](../../references/init-system-config.md) page.
```

Comment:
I would say it differently to stress that one doesn't have to use `tedge-services` and that a user can provide its own configuration.